index.js

```diff
@@ -248,6 +248,9 @@ Iso8583.prototype.encodeField = function(fieldName, fieldValue) {
         fieldSize = 0;
     } else if (builder && builder[0] && builder[0].binhex) {
         fieldSize = Buffer.byteLength(fieldValue, 'hex');
+        if (fieldSize % 8) {
```

Comment:
I do not think this is correct.
`fieldSize = Buffer.byteLength('0' + fieldValue, 'hex');` seems to be the proper approach

Reply:
Yes you are right, I think length have to be multiply of 8 but it is not true. Let me check and change all the flow.